### DISCUSSION TITLE
How to build ffcv from source?

### DISCUSSION THREAD

**Discussion by samuela** [libffcv/ffcv#157] [Q&A] [answered]
How does one go about building ffcv from source?

**Comment by GuillaumeLeclerc** [comment, marked as answer]
Hi @samuela.

Once you have the conda dependencies installed (or installed through using your OS package manager). It's as simple as cloning the repository and do (in the cloned folder):

```
pip install -e .
```
If you think it's a good thing to add it to the documentation/readme. Feel free to add a pull request.
